docs/feature/connectivity/index.md

```diff
@@ -1,85 +1,608 @@
-{tags-primary}`Import`
-{tags-primary}`Export`
-{tags-primary}`Extract`
-{tags-primary}`Load`
-{tags-primary}`ETL`
 
+:::{rubric} Overview
+:::
+All the CrateDB connectivity options at a glance.
 
-## Interfaces and Protocols
+You have a variety of options to connect to CrateDB, and to integrate it with
+off-the-shelve, 3rd-party, open-source, and proprietary applications.
 
-CrateDB supports both the [HTTP protocol] and the [PostgreSQL wire protocol],
+:::{rubric} About
+:::
+CrateDB supports both the HTTP protocol and the PostgreSQL wire protocol,
 which ensures that many clients that work with PostgreSQL, will also work with
 CrateDB.
 
-Through corresponding drivers, CrateDB is compatible with [ODBC],
-[JDBC], and other database API specifications.
-By supporting [SQL], CrateDB is compatible with many standard database
+Through corresponding drivers, CrateDB is compatible with JDBC, ODBC,
+and other database API specifications.
+By supporting SQL, CrateDB is compatible with many standard database
 environments out of the box.
 
-- [CrateDB HTTP interface]
-- [CrateDB PostgreSQL interface]
-- [CrateDB SQL protocol]
-
-To learn more, please refer to the documentation sections about supported
-client drivers, libraries, and frameworks, and corresponding tutorials.
-
-
-## Drivers and Integrations
+:::{rubric} Details
+:::
 
 CrateDB provides plenty of connectivity options with database drivers,
 applications, and frameworks, in order to get time series data in and
 out of CrateDB, and to connect to other applications.
 
-- [Drivers and Integrations]
-- [](inv:crate-clients-tools#connect)
-- [](inv:crate-clients-tools#df)
-- [](inv:crate-clients-tools#etl)
-- [](inv:crate-clients-tools#metrics)
+To learn more, please refer to the documentation sections and hands-on
+tutorials about supported client drivers, libraries, and frameworks,
+and how to configure and use them with CrateDB optimally.
+::::
 
-## Tutorials
 
-Hands-on tutorials about CrateDB fundamentals about data I/O, as well as about
-properly configuring and connecting relevant 3rd-party software components to
-work optimally with CrateDB.
+::::{grid-item}
+:class: rubric-slim
+:columns: auto 3 3 3
 
-:::{rubric} Overview
+```{rubric} Reference Manual
+```
+- [HTTP interface]
+- [PostgreSQL interface]
+- [SQL query syntax]
+
+```{rubric} Protocols and API Standards
+```
+- [HTTP protocol]
+- [PostgreSQL wire protocol]
+- [JDBC]
+- [ODBC]
+- [SQL]
+::::
+
+:::::
+
+
+
+## Synopsis
+
+:::{tip}
+The [CrateDB Examples] repository also includes a collection of
+clear and concise examples how to work with CrateDB.
 :::
-- [](#integrate)
-- [](#etl)
-- [](#integrate-metrics)
-- [](#performance)
+In order to provide a CrateDB instance for testing purposes, use, for
+example, Docker.
+```shell
+docker run --rm -it --publish=4200:4200 --publish=5432:5432 crate/crate:nightly
+```
+
+
+::::::{tab-set}
+
+:::::{tab-item} C# / Npgsql
+For more information see [Npgsql] documentation.
+
```

Comment:
That's the "tabbed" variant. We think it will not scale, and nesting tabs is also debatable.

Reply:
I agree, on a small screen (14 in laptop) it is not an optimal solution